bin/resource_management/update_resource_format.py

```diff
@@ -0,0 +1,31 @@
+import sys
+import csv
+import ckanapi
+"""

```

Comment:
docstrings need to be the first statement in a module to be accessible with `__doc__`